httpx/_models.py

```diff
@@ -140,6 +140,8 @@ def raw(self) -> typing.Tuple[bytes, bytes, typing.Optional[int], bytes]:
 
     @property
     def is_ssl(self) -> bool:
+        message = 'URL.is_ssl() is pending deprecation. Use url.scheme == "https"'
+        warnings.warn(message, DeprecationWarning)
```

Comment:
Note: we have a handy util in `_utils.py` so we don't have to remember which warning category should be passed (planning to issue a PR to update all usage to this shortly!).

```suggestion
        warn_deprecated(message)
```